src/main/java/graphql/ExecutionResult.java

```diff
@@ -11,6 +11,13 @@
 
     List<GraphQLError> getErrors();
 
-    Map<Object,Object> getExtensions();
+    Map<Object, Object> getExtensions();
```

Comment:
I think we should remove this: `graphql-java` doesn't support it currently and `ExecutionResult` should be immutable imho.